### PR TITLE
added good to go tick to homepage and added testimonial

### PR DESCRIPTION
**Pull request by oscarpinchen**
(cherry picked from commit d71b8b0c28a9371c38b103493269189ed3c8d984)

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -263,6 +266,30 @@ img {
   margin-bottom: 140px;
 }
 
+/* 'Good-To-Go' styling*/
```
Comment (supert56) [inline assets/css/styles.css:269]: Do we need these comments?

Comment (oscarpinchen) [reply assets/css/styles.css:269]: Yes

File: index.html [hunk 2]
```diff
@@ -55,6 +55,9 @@ <h1>Tour of Cambridge</h1>
             <a class="btn-pink" id="homepage_btn" href="/about">Find out more</a>
           </div>
       </div>
+      
+        <img class="good-to-go" src="../supert56.github.io/assets/images/GoodToGo.png" alt="Government 'Good-to-go' green tick">
```
Comment (supert56) [inline index.html:59]: You can just use `/assets/images/GoodToGo.png`. I think

**Review by supert56 — APPROVED** [review]
👍

**Pull request merged**